app/public/views/runs/runsPerPeriod/table/row.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { reduceSerialIf } from '../../../../utils/utils.js';
+import detectorIcon from '../../../../components/detectors/detectorIcon.js';
+import { RCT } from '../../../../config.js';
+
+export default function row(
+    model, visibleFields, data, item, cellsSpecials, index, runs, detectors,
+) {
```

Comment:
as `detectors` is instance of `class Detectors` is should be named differently, because it suggests it is array like `runs`. The same applies for other occurrences of this argument/variable

Reply:
not now